src/main/java/fc/server/palette/_common/config/SecurityConfig.java

```diff
@@ -58,6 +62,19 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
         return http.build();
     }
 
+    @Bean
+    CorsConfigurationSource corsConfigurationSource() {
+        CorsConfiguration config = new CorsConfiguration();
+        UrlBasedCorsConfigurationSource source = new UrlBasedCorsConfigurationSource();
+        config.setAllowCredentials(true);
+        config.addAllowedOrigin("*");
```

Comment:
`
When allowCredentials is true, allowedOrigins cannot contain the special value "*" since that cannot be set on the "Access-Control-Allow-Origin" response header. To allow credentials to a set of origins, list them explicitly or consider using "allowedOriginPatterns" instead.`
- setAllowCredentials(true) 와 addAllowedOrigin("*") 는 보안상의 문제로 같이 쓸 수 없다고 하네요
- allowedOriginPatterns("*") 같은 경우는 된다고 합니다

Reply:
추가로 `Assignees` 과 `Labels` 설정도 해주세요!

Reply:
allowedOriginPatterns("*") 로 바꿔 다시 푸시하겠습니다!